docs/application/native/guides/ui/tizen-ws-shell.md

```diff
@@ -0,0 +1,155 @@
+# Tizen Window System Shell
+
+
+The TZSH is an interface for manipulating windows of the sytem GUI services such as the quickpanel and the softkey. It provides C libraries that allow you to control windows of system GUI services and get notifications about the state changes of each window within an application.
+
+Since, the system GUI service runs different process, all of operations of the TZSH are performed using the inter-process communication (IPC). It is needed to enable communication between system GUI service processes and separated user application process using the display server. The following figure illustrates a simple software architecture of the TZSH:
+
+**Figure: Layer diagram for the Tizen window system shell**
+
+![Layer diagram for the Tizen window system shell](./media/tzsh_arch.png)
+
+To support each system GUI service in Tizen, the TZSH provides the following libraries:
+
+- **tzsh-quickpanel**: For the quickpanel window which shows notifications and system setup widgets.
+
+- **tzsh-softkey**: For the softkey window which shows the device back and home buttons.
+
+Not all applications require to use the TZSH's functionalities. However, in some cases, certain applications may require to perform manipulation of system GUI service's window. For example, media player application needs to close the quickpanel window during playback of video. In this case, you can use the tzsh-quickpanel library.
+
+
+## Prerequisites
+
+To use the functions and structures of the tzsh-quickpanel API (in [mobile](../../api/mobile/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html) and [wearable](../../api/wearable/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html) applications), include the `<tzsh_quickpanel.h>` header file in your application:
+
+```
+#include <tzsh_quickpanel.h>
+```
+
+
+## Creating TZSH Quickpanel Handler
```

Comment:
@nawab-samsung Don't we need to change the form of this title as others?

Reply:
I've changed it to 'Create TZSH Quickpanel Handler'.